Snippets/Snippets_4/Encryption/Configuration/encryption.config.xml

```diff
@@ -6,12 +6,11 @@
       name="RijndaelEncryptionServiceConfig"
       type="NServiceBus.Config.RijndaelEncryptionServiceConfig, NServiceBus.Core"/>
   </configSections>
-  <RijndaelEncryptionServiceConfig 
-    Key="gdDbqRpQdRbTs3mhdZh9qCaDaxJXl+e6">
+  <RijndaelEncryptionServiceConfig Key="gdDbqRpQdRbTs3mhdZh9qCaDaxJXl+e6" KeyIdentifier="B">
```

Comment:
since this will render inline can we newline the attributes to reduce dodgy wrapping in the html?